### ISSUE TITLE
Combobox: WCAG issue 4.1.2 found with ARC toolkit

### ISSUE THREAD

**Issue by Magnusrm** [digdir/designsystemet#2552]
### Version number

0.63.0 (and the version used in https://theme.designsystemet.no/)

### Description of the bug

When running ARC toolkit accessibility test on the Combobox component a violation is found.
![image](https://github.com/user-attachments/assets/6e5f9330-4bab-4777-a858-5b4e7a578a47)
<img width="371" alt="image" src="https://github.com/user-attachments/assets/cccc5423-e768-4970-954e-52eb91601d0f">
<img width="406" alt="image" src="https://github.com/user-attachments/assets/9ec1e779-b28a-4ba2-9f9e-745bc40e04b9">

### Steps To Reproduce

1. Go to https://theme.designsystemet.no/
2. Open devtools and run ARC toolkit: https://chromewebstore.google.com/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce
3. find the violation "Aria attribute value is incorrect" on the combobox that chooses a destination under the heading "Hvor skal du reise?"

### Additional Information

This issue was reported to Team Apps from UDIR as part of the WCAG tests they performed in preparation of the "Lærlingundersøkelsen" og senere "Elevundersøkelsen". They are going live next week and it would be ideal if we could get a fix for this as soon as possible in order to go live with as few WCAG violations as possible. @RonnyB71 @mimarz 

Related issues:
- https://github.com/Altinn/app-frontend-react/issues/2517#event-14477079311

**Comment by Barsnes** [comment]
Closing this as completed in release `0.63.1`
